Cargo.toml

```diff
@@ -24,7 +24,7 @@ frame-executive = { path = "pallets/executive" }
 frame-system-rpc-runtime-api = { path = "pallets/system/rpc/runtime-api" }
 frame-system-benchmarking = { path = "pallets/system/benchmarking" }
 
-# Polygon Primitives
+# DA Primitives
 da-primitives = { version = "0.4", git="https://github.com/maticnetwork/avail-core", tag = "da-primitives/v0.4.5" }
```

Comment:
We need to change the cargo refs from `maticnetwork` repos to `availproject` repos, right?

Reply:
Done `maticnetwork` references were replaced.